src/software/thunderscope/thunderscope_main.py

```diff
@@ -317,6 +339,7 @@
             estop_mode=estop_mode,
             estop_path=estop_path,
             enable_radio=args.enable_radio,
+            sslvision_address=args.sslvision_multicast_address,
```

Comment:
if `args.launch_gc` exists, then we should have a `gamecontroller.get_referee_port()` call somewhere here